lib/tictactoe.rb

```diff
@@ -24,8 +26,8 @@ def show_board
   end
 
   def play_turn
-    position = @player.selection
-    @board.place_token(position, "X")
+    position = @player.selection(@board)
+    @board.place_token(position, @board.next_token)
```

Comment:
Do we need to pass in the board's next token? 

https://pragprog.com/articles/tell-dont-ask

Reply:
next_token is the token to place on the board.  I guess we should set to a variable before passing in?

Reply:
If the board is responsible for determining the next token, and placing the token, do we need to pass the next token in when we place it?

Reply:
Nice insight